venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -35,51 +35,276 @@ def process(client, note, invitation):
+        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Revision',
+        super = 'OpenReview.net/Support/-/Revision',
         invitees = readers,
         reply = {
             'forum': forum.id,
-            'replyto': forum.id,
+            'referent': forum.id,
             'readers' : {
                 'description': 'The users who will be allowed to read the above content.',
                 'values' : readers
             }
         },
-        signatures = ['OpenReview.net/Support']
+        signatures = [conference.get_program_chairs_id()]
     ))
 
+    recruitment_email_subject = '[{Abbreviated_Venue_Name}] Invitation to serve as {invitee_role}'.replace('{Abbreviated_Venue_Name}', conference.get_short_name())
+    recruitment_email_body = '''Dear {name},
+
+You have been nominated by the program chair committee of {Abbreviated_Venue_Name} to serve as {invitee_role}. As a respected researcher in the area, we hope you will accept and help us make {Abbreviated_Venue_Name} a success.
+
+You are also welcome to submit papers, so please also consider submitting to {Abbreviated_Venue_Name}.
+
+We will be using OpenReview.net with the intention of have an engaging reviewing process inclusive of the whole community.
+
+To ACCEPT the invitation, please click on the following link:
+
+{accept_url}
+
+To DECLINE the invitation, please click on the following link:
+
+{decline_url}
+
+Please answer within 10 days.
+
+If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
+
+If you have any questions, please contact info@openreview.net.
+
+Cheers!
+
+Program Chairs'''.replace('{Abbreviated_Venue_Name}', conference.get_short_name())
+
     if (forum.content['Area Chairs (Metareviewers)'] == "Yes, our venue has Area Chairs") :
-        ac_recruitment_invitation = client.post_invitation(openreview.Invitation(
-            id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Area_Chair_Recruitment',
-            super = 'OpenReview.net/Support/-/Area_Chair_Recruitment',
+        metareview_stage_invitation = client.post_invitation(openreview.Invitation(
+            id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Meta_Review_Stage',
+            super = 'OpenReview.net/Support/-/Meta_Review_Stage',
             invitees = readers,
             reply = {
                 'forum': forum.id,
-                'replyto': forum.id,
+                'referent': forum.id,
                 'readers' : {
                     'description': 'The users who will be allowed to read the above content.',
                     'values' : readers
                 }
             },
-            signatures = ['OpenReview.net/Support']
+            signatures = [conference.get_program_chairs_id()]
         ))
+        recruitment_invitation = client.post_invitation(openreview.Invitation(
```

Comment:
Can we define the invitation once and parametrize the value-radio array?

Reply:
Agreed. Changes have been made in accordance.